README.md

```diff
@@ -49,6 +49,8 @@ The `createNav` function takes an object of options with the following property:
 
 - `breakpoint`: The point, in pixels, at which the navigation switches between desktop and mobile layouts. The default is `620px`, which is meant to reflect the default value of `$breakpoint-navigation-threshold` in Vanilla (see [Vanilla's breakpoint documentation](https://vanillaframework.io/docs/settings/breakpoint-settings)).
 
+- `mobileContainerSelector` and `desktopContainerSelector`: Selectors (can be 'id' or 'class') of where to attach the mobile and desktop views of the global-nav. This will also circumvent the default eventListeners being attached as they are dependent on the default structure, so custom JS will be nessacary. Also in this case, the the class `global-nav` should still be used to indicate the position of the 'All Canonical' tab button. If ony one is used it will render the default global nav.
```

Comment:
```suggestion
- `mobileContainerSelector` and `desktopContainerSelector`: Selectors (can be 'id' or 'class') of where to attach the mobile and desktop views of the global-nav. This will also circumvent the default eventListeners being attached as they are dependent on the default structure, so custom JS will be nessacary. Also in this case, the class `global-nav` should still be used to indicate the position of the 'All Canonical' tab button. If ony one is used it will render the default global nav.
```